modules/ai/assets/js/editor/pages/form-media/index.js

```diff
@@ -63,9 +65,11 @@ const FormMedia = ( {
 	};
 
 	const editImageInitialData = getControlValue() === additionalOptions?.defaultValue ? {} : getControlValue();
+	const isGenerateScreen = ! editImageInitialData?.id;
```

Comment:
Not sure that I get this, in which scenario the history should not be presented?

Reply:
It's something tbd within a team. 
Just for context: the first iteration will only provide history items for image generation and the BE returns only them. I thought it might be confusing for users to have the button on the editing screens if we can't provide editing history events for now.